network/interface.go

```diff
@@ -56,6 +57,8 @@ type Transactions interface {
 	// WithPersistency returns a SubscriberOption for persistency. It allows the DAG KVStore to be used as persistent store for notifications.
 	// The notifications will then have ACID properties
 	WithPersistency() SubscriberOption
+	// ServiceDiscovery should be called by the VDR to let the network know it has processed and verified a document (update) for the DID.
+	ServiceDiscovery(updatedDID did.DID)
```

Comment:
I'd expect a verb in a function (except when it just returns something), e.g. `DiscoveryServices()`

Reply:
changed to `DiscoverServices()`